zach_brady/mid-term/main.rb

```diff
@@ -0,0 +1,103 @@
+require 'pry'
+require_relative 'lib/animal'
+require_relative 'lib/client'
+require_relative 'lib/shelter'
+
+henry = Animal.new("Henry Miller", "Cat")
+me = Client.new("Zach", 26)
+$sugar_mutts = Shelter.new("Sugar Mutts")
+
+while true
+    puts
+    puts "Welcome to the Sugar Mutts Shelter Directory. "
+    puts "  - (a) add an animal"
+    puts "  - (b) add a client"
+    puts "  - (c) list all animals"
+    puts "  - (d) list all clients"
+    puts "  - (e) adoption"
+    puts "  - (f) return pet (you asshole.)"
+    puts "  - (q) quit"
+    puts
+
+    answer = gets.chomp.downcase
+    puts
+
+    case answer
```

Comment:
one odd difference with JS is that case is actually slower than if/else, interestingly enough. Totally the opposite in JavaScript. I [found that out](http://midwire.github.io/blog/2011/08/26/ruby-performance-case-vs-if-elsif/) one time when I had a freaking massive if else in a JS project that too long enough to irritate a patient user. helped a tiny bit to use case.